Gulpfile.js

```diff
@@ -38,14 +41,25 @@ function compile(advanced = false) {
         .pipe(sourcemaps.init())
         .pipe(closureCompiler(options))
         .pipe(sourcemaps.write('/'))
-        .pipe(gulp.dest('dist'));
+        .pipe(gulp.dest('dist/js'));
+}
+
+function cssMinify() {
+    return gulp.src('src/components/**/*.css')
+        .pipe(concatCss('css/erste.css'))
+        .pipe(sourcemaps.init())
+        .pipe(cleanCss())
+        .pipe(rename({ suffix: '.min' }))
+        .pipe(sourcemaps.write('/'))
+        .pipe(gulp.dest('dist'))
 }
 
 gulp.task('clean', () => del(['dist/*']));
 gulp.task('compile:simple', compile.bind(null, false));
 gulp.task('compile:advanced', compile.bind(null, true));
+gulp.task('css:min', () => { cssMinify() });
```

Comment:
I believe you can directly pass cssMinify as a parameter instead of an anonymous function.